### ISSUE TITLE
Might be helpful to remind users to add Radicle keys to ssh-agent

### ISSUE THREAD

**Issue by keepsimple1** [radicle-dev/heartwood#468]
https://github.com/radicle-dev/heartwood/blob/fc724a2333040ced38f87aa4c70149ffab095bb7/radicle-cli/examples/workflow/4-patching-contributor.md?plain=1#L14-L19

If the user was not in the same shell of `rad auth` earlier,  they might forgot to `ssh-add` their keys to `ssh-agent` again. If that's the case, they likely encounter an error during `git commit`, something like this: 

`error: Load key "/tmp/.git_signing_key_tmpBQTgrv": error in libcrypto?. `

I think it's worth it to add a reminder here (or somewhere else) for the user to add their `rad self --ssh-key` to `ssh-agent` (one time, or permanently).   (I found out this in hard way ;-).

**Comment by cloudhead** [comment]
The thing is, the default path users take is not to use the radicle key for signing. So in that case there shouldn't be any issue. It's only if you configure radicle for signing (or any ssh key really) that you may encounter that issue.

**Comment by keepsimple1** [comment]
> the default path users take is not to use the radicle key for signing.

That means my experience was not typical. The first repo I tried is `radicle-team` repo and I followed the instructions to create & use the radicle keys. If I understand, are you saying that we don't expect most users to configure radicle or any ssh keys for signing their repos,  and if they do, they should know what they are doing with ssh-agent already?
